pkg/capabilities/pb/capabilities.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.31.0
-// 	protoc        v4.25.1
+// 	protoc        v4.24.4
```

Comment:
regression here and elsewhere

Reply:
yeah, I used make generated, I'm surprised it doesn't enforce the tool version.  Looking into it